cluster/validation.go

```diff
@@ -39,8 +40,18 @@ func (c *Cluster) ValidateCluster() error {
 }
 
 func validateAuthOptions(c *Cluster) error {
-	if c.Authentication.Strategy != DefaultAuthStrategy {
-		return fmt.Errorf("Authentication strategy [%s] is not supported", c.Authentication.Strategy)
+	for S, v := range c.AuthnStrategies {
+		if !v {
+			continue
+		}
+		s := strings.ToLower(S)
+		logrus.Info("Has authn strategy: ", s)
```

Comment:
can we add this as a debug instead of info

Reply:
cleaned up variable names & removed logging